.github/workflows/kubeapps-general.yaml

```diff
@@ -609,10 +602,9 @@ jobs:
       - push_images
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
       - uses: ./.github/actions/srp-source-provenance
         with:
-          SRP_CLI_VERSION: ${{env.SRP_CLI_VERSION}}
+          SRP_CLI_VERSION: ${SRP_CLI_VERSION}
```

Comment:
I think this is an unintentional change - but should go away if you merge main? (I saw @beni0888 fix this in main earlier today - adding the line that your PR is reverting here by accident).

Reply:
yes, don't worry about it. I am in a testing phase